keyboards/palette1202/readme.md

```diff
@@ -0,0 +1,24 @@
+# Palette1202
+
+---
+
```

Comment:
You don't really *need* this horizontal rule here; GitHub adds a thin bottom border to the header you have on Line 1, so you could delete Lines 3-4 if you wish.

Reply:
I'll remove this, thanks!